install.ps1

```diff
@@ -315,7 +315,26 @@ function Install-Mondoo {
            $login_params = $login_params + @("--api-proxy", "$Proxy")
         }
         $program = "$Path\cnspec.exe"
-        & $program $login_params
+
+        # Cache the error action preference
+        $backupErrorActionPreference = $ErrorActionPreference
+        $ErrorActionPreference = "Continue"
+
+        # Capture all output from cnspec
+        $output = (& $program $login_params 2>&1)
```

Comment:
I think we do not need the redirect, if we set `$ErrorActionPreference = "Continue"`

Reply:
As discussed with @atomic111 not doing this leads to powershell writing the output in red. So the redirect is the right way